OptionSelector/DetailCategoryTableViewController.m

```diff
@@ -0,0 +1,55 @@
+//
+//  DetailCategoryTableViewController.m
+//  OptionSelector
+//
+//  Created by Jason Wang on 8/15/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "DetailCategoryTableViewController.h"
+
+@interface DetailCategoryTableViewController ()
+
+@end
+
+@implementation DetailCategoryTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+}
+
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
```

Comment:
Style: The [default number of sections](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UITableViewDataSource_Protocol/#//apple_ref/occ/intfm/UITableViewDataSource/numberOfSectionsInTableView:) in a table view is 1, and this method is optional, so you can remove it as well.